### ISSUE TITLE
change tests to use models as fixtures

### ISSUE THREAD

**Issue by dbrakenhoff** [ArtesiaWater/nlmod#21]
hopefully make testing a bit more efficient?

**Comment by martinvonk** [comment]
I think what would also help is to create a logical pipeline for the tests. This might save time by making the tests more specific, using workflows that are dependent on one another. 

https://github.blog/enterprise-software/ci-cd/build-ci-cd-pipeline-github-actions-four-steps/

**Comment by martinvonk** [comment]
Yes: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/storing-and-sharing-data-from-a-workflow
